recipe/meta.yaml

```diff
@@ -1,47 +1,50 @@
 {% set name = "libblasfeo" %}
-{% set version = "0.1.3" %}
-{% set build = 3 %}
+{% set version = "0.1.4" %}
+{% set build = 0 %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   - url: https://github.com/giaf/blasfeo/archive/refs/tags/{{ version }}.tar.gz
-    sha256: c33eb6467a90d6075a8db34e5ab239ecdda38c5261ae096def122fe7a0f98780
-    patches:
-      - skip_blas_tests.patch
+    sha256: a948e2cab85d0280653d29c709fd7f44ba0829fb09327c29c28d070492230386
 
 build:
-  # windows and macos are supported on the master branch, as soon as a new version is released
-  # they can be packaged (see https://github.com/giaf/blasfeo/issues/181)
-  skip: True  # [win or osx]
-  number: {{ build }}          # [not (linux and x86_64)]
-  number: {{ build + 100 }}    # [linux and x86_64 and microarch_level == 1]
-  number: {{ build + 300 }}    # [linux and x86_64 and microarch_level == 3]
+  number: {{ build }}  # [not x86_64]
+  number: {{ build + 100 }}  # [x86_64 and microarch_level == 1]
+  number: {{ build + 300 }}  # [x86_64 and microarch_level == 3]
+  number: {{ build + 400 }}  # [x86_64 and microarch_level == 4]
   run_exports:
     - {{ pin_subpackage(name, max_pin='x.x.x') }}
   script_env:
     # See https://github.com/giaf/blasfeo?tab=readme-ov-file#supported-computer-architectures
-    # X64_INTEL_CORE is the TARGET setting more similar to nocona, so we used that for
-    # no microarch_level and microarch_level == 1
+    # and https://clang.llvm.org/docs/UsersManual.html#x86
+    # X64_INTEL_CORE is the TARGET setting more similar to nocona,
+    # so we used that for no microarch_level and microarch_level == 1
     # microarch_level == 3 has AVX and AVX2, so it is basically TARGET X64_INTEL_HASWELL
-    # Commented as a workaround for https://github.com/conda-forge/blasfeo-feedstock/pull/9,
-    # we can re-enable optimized code once new release of blasfeo is released
-    #- BLASFEO_TARGET=GENERIC  # [not x86_64]
-    #- BLASFEO_TARGET=X64_INTEL_CORE  # [(not linux) and x86_64]
-    #- BLASFEO_TARGET=X64_INTEL_CORE  # [linux and x86_64 and microarch_level == 1]
-    #- BLASFEO_TARGET=X64_INTEL_HASWELL  # [linux and x86_64 and microarch_level == 3]
-    - BLASFEO_TARGET=GENERIC
+    # microarch_level == 4 has AVX512F, so it is basically TARGET X64_INTEL_SKYLAKE_X
+    - BLASFEO_TARGET=GENERIC  # [not (x86_64 or aarch64 or arm64)]
+    - BLASFEO_TARGET=X64_INTEL_CORE  # [x86_64 and microarch_level == 1]
+    - BLASFEO_TARGET=X64_INTEL_HASWELL  # [x86_64 and microarch_level == 3]
+    - BLASFEO_TARGET=X64_INTEL_SKYLAKE_X  # [x86_64 and microarch_level == 4]
+    # Modern arm processors (e.g. X Elite) have cache sizes more similar to the
+    # M1 from Apple, thus we set TARGET to ARMV8A_APPLE_M1
+    - BLASFEO_TARGET=ARMV8A_APPLE_M1  # [aarch64 or arm64]
```

Comment:
I am bit dubious on this, at least for Linux. Is this compatible with embedded arm boards, for example Raspberry PI or Jetson? We frequently use conda packages there.

Reply:
That's actually a good point. As far as I can see in the blasfeo source code, the only difference is in the config of the L1 and L2 cache. Hence, it probably makes sense to use ARMV8A_ARM_CORTEX_A76 (Raspberry Pi) on Linux and ARMV8A_APPLE_M1 everywhere else. For Windows, the higher end chips (e.g. X Elite) are probably the more accurate targets.